containers-tests/tests/map-strictness.hs

```diff
@@ -82,6 +94,35 @@ pInsertLookupWithKeyValueStrict f k v m
                      not (isBottom $ M.insertLookupWithKey (const3 1) k bottom m)
     | otherwise    = isBottom $ M.insertLookupWithKey (apply3 f) k bottom m
 
+#if __GLASGOW_HASKELL__ >= 806
+pStrictFoldr' :: Map Int Int -> Property
+pStrictFoldr' m = ioProperty $ isNothing <$> noThunks [] r
```

Comment:
Could you turn this `isNothing <$> noThunks` pattern into a little helper?

Reply:
Done